examples/vmseries_ha/README.md

```diff
@@ -0,0 +1,284 @@
+ - `linux_vms.<vm-name>.linux_disk_size`
+
+1. Apply the terraform code:
+
+```
+terraform init
+terraform apply 
+```
+
+4. Check the output plan and confirm the apply.
+
+5. Check the successful application and outputs of the resulting infrastructure:
+
+```
+Apply complete! Resources: 96 added, 0 changed, 0 destroyed. (Number of resources can vary based on how many instances you push through tfvars)
+
+Outputs:
+
+lbs_internal_ips = {
+  "external-lb" = "<EXTERNAL_LB_PUBLIC_IP>"
+}
+lbs_internal_ips = {
+  "internal-lb" = "10.10.12.5"
+}
+linux_vm_ips = {
+  "spoke1-vm" = "192.168.1.2"
+  "spoke2-vm" = "192.168.2.2"
+}
+vmseries_private_ips = {
+  "fw-vmseries-01" = {
+    "0" = "10.10.11.2"
+    "1" = "10.10.10.2"
+    "2" = "10.10.12.2"
+    "3" = "10.10.13.2"
+  }
+  "fw-vmseries-02" = {
+    "0" = "10.10.11.3"
+    "1" = "10.10.10.3"
+    "2" = "10.10.12.3"
+    "3" = "10.10.13.3"
+  }
+}
+vmseries_public_ips = {
+  "fw-vmseries-01" = {
+    "0" = "<UNTRUST_PUBLIC_IP>"
+    "1" = "<MGMT_PUBLIC_IP>"
+  }
+  "fw-vmseries-02" = {
+    "0" = "<UNTRUST_PUBLIC_IP>"
+    "1" = "<MGMT_PUBLIC_IP>"
+  }
+}
+```
+
+
+## Post build
+
+Connect to the VM-Series instance(s) via SSH using your associated private key and check if the bootstrap process if finished successfuly and then set a password :
+  - Please allow for up to 10-15 minutes for the bootstrap process to finish
+  - The key output you should check for is "Auto-commit Successful"
+
+```
+ssh admin@x.x.x.x -i /PATH/TO/YOUR/KEY/id_rsa
+Welcome admin.
+admin@PA-VM(active)> show system bootstrap status
+
+Bootstrap Phase               Status         Details
+===============               ======         =======
+Media Detection               Success        Media detected successfully
+Media Sanity Check            Success        Media sanity check successful
+Parsing of Initial Config     Successful     
+Auto-commit                   Successful
+
+admin@PA-VM(active)> configure
+Entering configuration mode
+[edit]                                                                                                                                                                                  
+admin@PA-VM(active)# set mgt-config users admin password
+Enter password   : 
+Confirm password : 
+
+[edit]                                                                                                                                                                                  
+admin@PA-VM(active)# commit
+Configuration committed successfully
+```
+
+## Check access via web UI
+
+Use a web browser to access `https://<MGMT_PUBLIC_IP>` and login with admin and your previously configured password.
+
+## Change the public Loopback public IP Address
+
+For the VM-Series that are backend instance group members of the public-facing loadbalancer - go to Network -> Interfaces -> Loopback and change the value of `1.1.1.1` with the value from the `EXTERNAL_LB_PUBLIC_IP` from the terraform outputs.
```

Comment:
In TF outputs it's used name `lbs_external_ips` for map, which contains key `external-lb`, so maybe could we rephrase it into: 

```suggestion
For the VM-Series that are backend instance group members of the public-facing loadbalancer - go to Network -> Interfaces -> Loopback and change the value of `1.1.1.1` with the value of object `external-lb` from the `lbs_external_ips` in Terraform outputs.
```